lib/seek/acts_as_asset/content_blobs.rb

```diff
@@ -5,7 +5,7 @@ module ContentBlobs
       module ClassMethods
         extend ActiveSupport::Concern
         included do
-          after_destroy :mark_deleted_content_blobs
+          after_destroy :mark_deleted_content_blobs, unless: -> { self.class == SampleType }
```

Comment:
Because this test fails:
https://github.com/seek4science/seek/blob/8463e2774e021c3d4c675c1bf3c68611644bbd31/test/unit/sample_type_test.rb#L517-L536

```
Error:
SampleTypeTest#test_dependant_destroy_content_blob:
ActiveRecord::ActiveRecordError: cannot update a destroyed record
    lib/seek/acts_as_asset/content_blobs.rb:57:in `block in mark_deleted_content_blobs'
    lib/seek/acts_as_asset/content_blobs.rb:56:in `each'
    lib/seek/acts_as_asset/content_blobs.rb:56:in `mark_deleted_content_blobs'
    test/unit/sample_type_test.rb:459:in `block (5 levels) in <class:SampleTypeTest>'
    test/unit/sample_type_test.rb:458:in `block (4 levels) in <class:SampleTypeTest>'
    test/unit/sample_type_test.rb:457:in `block (3 levels) in <class:SampleTypeTest>'
    app/models/user.rb:271:in `with_current_user'
    test/unit/sample_type_test.rb:449:in `block (2 levels) in <class:SampleTypeTest>'
    test/test_helper.rb:77:in `with_config_value'
    test/unit/sample_type_test.rb:448:in `block in <class:SampleTypeTest>'
```

I assume it has something to do with this line:
https://github.com/seek4science/seek/blob/8463e2774e021c3d4c675c1bf3c68611644bbd31/lib/seek/templates/sample_type_template_concerns.rb#L10

I didn't want to mess too much with it, so I skipped sample types to be safe.

Reply:
yes, that would be why, I think would be better to remove the `depedent: destroy` and keep the other behaviour consistent